styles/main.css

```diff
@@ -1,5 +1,9 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
 
+* {
+  box-sizing: border-box;
+}
+
 body {
```

Comment:
- [ ] I think it would be nice to reset the default margin of the body tag as it gives your page a grey margin around the screen. See the difference with the design below:
#### Your project
![image](https://user-images.githubusercontent.com/61377322/145518679-a819ec07-da25-485f-85c3-009c761b348a.png)

#### Figma design
![image](https://user-images.githubusercontent.com/61377322/145518731-21b5d55f-bab5-42e0-b9ae-e0b267b4a2c4.png)